example/dms.tf

```diff
@@ -1,41 +1,106 @@
-variable "source_database_name" {
-}
+module "test_dms" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-dms?ref=new-use"
```

Comment:
Ref should equal latest module. Maybe you could use Terraform docs to generate this or maybe just make it obvious it needs to change i.e. `<latest_module` or something similar.